release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -63,6 +63,17 @@ User-defined tags for Microsoft Azure was previously introduced as Technology Pr
 [id="ocp-4-14-web-console"]
 === Web console
 
+[id="ocp-4-14-openshift-cli"]
+=== OpenShift CLI (oc)
+
+.Support multi-arch OCI local images for catalogs with oc-mirror
```

Comment:
```suggestion
.Supporting multi-arch OCI local images for catalogs with oc-mirror
```

Here's the guideline:
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assemblymodule-titles-and-section-headings

Reply:
Ack.